rig/machine_control/machine_controller.py

```diff
@@ -396,6 +396,127 @@ def read(self, address, length_bytes, x, y, p=0):
         return connection.read(self.scp_data_length, self.scp_window_size,
                                x, y, p, address, length_bytes)
 
+    @ContextMixin.use_contextual_arguments()
+    def link_write(self, address, data, x, y, link):
+        """Write a bytestring to an address in memory on a neigbouring chip.
+
+        .. warning::
+
+            This function is intended for low-level debug use only and is not
+            optimised for performance nor intended for more general use.
+
+        This method allows you to instruct a monitor processor to send 'POKE'
+        nearest-neighbour packets to a neighbouring chip. These packets are
+        handled directly by the SpiNNaker router in the neighbouring chip,
+        potentially allowing advanced debug or recovery of a chip rendered
+        otherwise unreachable.
+
+        Parameters
+        ----------
+        address : int
+            The address at which to start reading the data. Only addresses in
```

Comment:
Of course ;) For some reason GitHub doesn't want to hide this comment... Should be fixed now!